### PR TITLE
Sort json output

### PR DESCRIPTION
**Pull request by adisos**
issue #157

### CODE REVIEW

File: pkg/vpcmodel/jsonOutput.go
Comment (zivnevo) [inline pkg/vpcmodel/jsonOutput.go:1]: Please fix the error message at the end of the file, which refers to `md` instead to `json`

File: pkg/vpcmodel/jsonOutput.go [hunk 1]
```diff
@@ -15,6 +16,20 @@ type connLine struct {
...
+func sortConnLines(connLines []connLine) {
+	sort.Slice(connLines, func(i, j int) bool {
+		if connLines[i].Src.Name() != connLines[j].Src.Name() {
+			return connLines[i].Src.Name() < connLines[j].Src.Name()
+		}
+		if connLines[i].Dst.Name() != connLines[j].Dst.Name() {
+			return connLines[i].Dst.Name() < connLines[j].Dst.Name()
+		}
+
+		return connLines[i].connStr < connLines[j].connStr
```
Comment (zivnevo) [inline pkg/vpcmodel/jsonOutput.go:31]: How can we have two lines with the same <src, dst>?
Seems to me like we don't really need `connStr`.

**Review by zivnevo — CHANGES_REQUESTED** [review]

File: pkg/vpcmodel/jsonOutput.go
Comment (adisos) [reply pkg/vpcmodel/jsonOutput.go:1]: Fixed.

File: pkg/vpcmodel/jsonOutput.go (hunk 1 above)
Comment (adisos) [reply pkg/vpcmodel/jsonOutput.go]: Right, fixed.

**Review by zivnevo — APPROVED** [review]
LGTM

**Pull request merged**